templates/intro/cant-find-me/.config

```diff
@@ -1,4 +1,5 @@
 __TPL_DB_PATH=/home/ctf/database/
 __TPL_WORD=secret
-__TPL_CONNECT_TO=ctf@141.85.224.118
+__TPL_CONNECT_TO=ctf@141.85.224.157:2242
 __TPL_PASSWORD=cant-find-me-0
+__TPL_PLATFORM_MESSAGE=Submit the flag on the `CTF platform <https://sis-ctf.security.cs.pub.ro/challenges?category=session-01-intro>`_. You need to create an account on the platform if you don't already have one.
```

Comment:
I'm not sure about having UPB specific links merged into master.
I think we can go forward for now, but we should think this through.

Reply:
@CostinCarabas , we agreed that we will fill the configuration files (`.config`) with UPB information that would then be updated by the people who fork.

I share you concern on providing UPB links. I'm OK with removing that link and then adding it to our `sis-cs.pub.ro` repository/fork. The repository will still be public though.

@raresvis , what do you think?

Reply:
I will merge this pull request. We agreed the config files will be UPB links and IP addresses so that's fine.

Reply:
When we discussed this issue, we agreed that we will use our own IPs. The fact is that we can't really do it any other way.